design-toolkit/components/src/components/query-builder/action-element.tsx

```diff
@@ -42,7 +34,7 @@ export function ActionElement({
   return (
     <Button
       isDisabled={disabled && !disabledTranslation}
-      onPress={handlePress}
+      onPress={() => handleOnClick()}
```

Comment:
curious why not this syntax?

```suggestion
      onPress={handleOnClick}
```